core/src/main/java/bio/terra/pearl/core/service/export/formatters/SurveyFormatter.java

```diff
@@ -114,6 +121,17 @@ protected String stripStudyPrefixes(String stableId) {
         return stableId.substring(stableId.lastIndexOf('_') + 1);
     }
 
+    public ItemExportInfo getItemExportInfo(ExportOptions exportOptions, String moduleName, List<SurveyQuestionDefinition> questionDefs)
```

Comment:
Might be helpful to have a JavaDoc comment, or some in-line comments explaining what's going on here. 
I am new to this code, but I am unclear on whether the `List<SurveyQuestionDefinition>` represents a list of versions for a specific question, or a list of questions in a survey.
Likewise, what a module represents is not clear from the context in this file, so someone new to this code has to switch context to refresh on that.
In short there are a handful of entities in play here, and it would be helpful to have better names for those entities and/or some hints in comments.